Sources/Wrappers/AngouriMath.Interactive/AggressiveOperators.fs

```diff
@@ -12,3 +13,47 @@ let ( / ) a b =
     ((parsed a) / (parsed b)).InnerSimplified
 let ( ** ) a b =
     ((parsed a).Pow(parsed b)).InnerSimplified
+let ( ~~~ ) a = 
+    (negation (parsed a)).InnerSimplified
+let ( ||| ) a b = 
+    (union (parsed a) (parsed b)).InnerSimplified
+let ( ^^^ ) a b = 
+    (intersect (parsed a) (parsed b)).InnerSimplified
+let ( = ) a b = 
+    (equality (parsed a) (parsed b)).InnerSimplified
+let ( <> ) a b = 
+    (negation (equality (parsed a) (parsed b))).InnerSimplified
+let ( > ) a b = 
+    (greater (parsed a) (parsed b)).InnerSimplified
+let ( < ) a b = 
+    (less (parsed a) (parsed b)).InnerSimplified
+let ( >= ) a b = 
+    (greaterOrEqual (parsed a) (parsed b)).InnerSimplified  
+let ( <= ) a b = 
+    (lessOrEqual (parsed a) (parsed b)).InnerSimplified
+let sin a = 
+    (sin (parsed a)).InnerSimplified
+let cos a = 
+    (cos (parsed a)).InnerSimplified
+let tan a = 
+    (tan (parsed a)).InnerSimplified
+let asin a = 
+    (asin (parsed a)).InnerSimplified
+let acos a = 
+    (acos (parsed a)).InnerSimplified
+let atan a = 
+    (atan (parsed a)).InnerSimplified
+let sinh a = 
+    (sinh (parsed a)).InnerSimplified
+let cosh a = 
+    (cosh (parsed a)).InnerSimplified
+let tanh a = 
+    (tanh (parsed a)).InnerSimplified
+let sqrt a = 
+    (sqrt (parsed a)).InnerSimplified
+let log logbase a = 
+    (log (parsed logbase) (parsed a)).InnerSimplified
+let log10 a = 
```

Comment:
Why are those in the PR? 🤔

Reply:
Oh, I wasn't sure if these were included as F# math functions. It looked to me like F# had their own implementations of these, so I overrode them to use the AngouriMath substitutes.

Reply:
But we already have them in AngouriMath.FSharp.Functions, don't we?

Reply:
Oh, right, I guess those would already override them. I'll remove these then.